pkg/network/node/node.go

```diff
@@ -471,7 +471,7 @@ func (node *OsdnNode) UpdatePod(pod kapi.Pod) error {
 	return err
 }
 
-func (node *OsdnNode) GetRunningPods(namespace string) ([]kapi.Pod, error) {
+func (node *OsdnNode) GetRunningPods(namespace string) (map[string]kapi.Pod, error) {
```

Comment:
add a quick docblock of what the return parameters are.

Reply:
Done